src/isar_turtlebot/turtlebot/turtlebot.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional
+
+from isar_turtlebot.models.turtlebot_status import Status
+from isar_turtlebot.ros_bridge import RosBridge
+from isar_turtlebot.turtlebot.taskhandlers.driveto import DriveToHandler
+from isar_turtlebot.turtlebot.taskhandlers.takeimage import TakeImageHandler
+from isar_turtlebot.turtlebot.taskhandlers.takethermalimage import (
+    TakeThermalImageHandler,
+)
+from isar_turtlebot.turtlebot.taskhandlers.taskhandler import TaskHandler
```

Comment:
Add the different `Handler` classes to the `__init__.py` file of the `TaskHandler` folder. Then they may all be imported from `isar_turtlebot.turtlebot.taskhandlers`

Reply:
Good idea. Done.